docs/_basic/listening_modals.md

```diff
@@ -69,3 +73,21 @@ app.view('view_b', async ({ ack, body, view, client }) => {
 
 });
 ```
+
+---
+
+##### Handling views on close
+
+💡 When listening for `view_closed` requests, you must pass an object containing `type: 'view_closed'` and the view `callback_id`. See below for an example of this:
+
+See the <a href="https://api.slack.com/surfaces/modals/using#modal_cancellations">API documentation</a> for more information about `view_closed.
```

Comment:
We could also probably use the git-flavored markdown syntax here (see the example above) for this link as well. Apologies, we're not consistent on that across documentation - trying to move links to that format!